aten/src/ATen/core/interned_strings.cpp

```diff
@@ -14,7 +14,7 @@
 namespace c10 {
 
 const std::string& domain_prefix() {
```

Comment:
This is used for some caffe2 ops.
There are test failures related to this change.
Look into TestOperators.test_c2_op for example (org.pytorch._caffe2 is now _caffe2)

Reply:
Yes, I'm looking into a way to distinguish caffe2 ops, cause this is prefixed to all custom ops for now. I'll look into it.

Reply:
domainString() is called when !node->kind().is_onnx() (line 333). 
Maybe you can separate this "if" into if caffe2 or aten call domainString, otherwise use an empty string for the domain